book/_config.yml

```diff
@@ -4,11 +4,12 @@
 title: QIIME 2 FAES Workshop Tutorial, January 2022
 author: The QIIME 2 Developers
 logo: logo.png
-copyright: 2022
+copyright: "2022"
 
 # See https://jupyterbook.org/content/execute.html
 execute:
-  execute_notebooks: cache
+  # force seems to be necessary for maintaining scope across chapters
+  execute_notebooks: force
```

Comment:
I'll take a closer look at this, but this doesn't sound right - the `execute_notebooks` config is for running code as notebook cells, and shouldn't have any relationship to the usage sphinx extension (my understanding is cell/notebook execution is essentially parallel to our sphinx extension).

Reply:
Okay, I see why you set this - while this setting is necessary for maintaining scope across chapters, it might be more clear to indicate that this is necessary for ensuring that _all usage examples_ are run at build time. Its more like "force is necessary for populating scope with all usage example blocks," if that makes any sense.